fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/DimensionRowSpec.groovy

```diff
@@ -12,18 +14,27 @@ class DimensionRowSpec extends Specification {
     String nonKeyFieldName = "nonKeyFieldName"
     DimensionField keyField = Mock(DimensionField)
     DimensionField nonKeyField = Mock(DimensionField)
+    DimensionRow testRow
 
     def setup() {
         keyField.name >> keyFieldFieldName
         nonKeyField.name >> nonKeyFieldName
+        testRow = new DimensionRow(keyField, [(keyField): keyValue, (nonKeyField): nonKeyValue])
     }
 
     def "healthy initialization returns correct state"() {
-        setup:
-        DimensionRow testRow = new DimensionRow(keyField, [(keyField): keyValue, (nonKeyField): nonKeyValue])
-
         expect:
         testRow.getKeyValue() == keyValue
         testRow.getRowMap() == [(nonKeyFieldName): nonKeyValue, (keyFieldFieldName): keyValue]
     }
+
+    def "json serialization into the correct format we expect"() {
+        setup:
```

Comment:
'correct format we expect' is redundant.  
Perhaps 'all expected fields and values are present in the dimension row serialization'